### ISSUE TITLE
Implement Windows+[Number] keys to launch apps and access windows

### ISSUE THREAD

**Issue by sandrock** [cairoshell/cairoshell#650]
**Is your feature request related to a problem? Please describe.**
On Windows shell, I heavily use the <kbd>Windows+[Number]</kbd> keyboard shortcut to launch apps.

> Exemple:  
Windows+1 helps me launch Firefox  
Windows+2 helps me launch Firefox Developer  
Windows+3 helps me launch Explorer  
Windows+4 helps me launch Audacious    
...

Also, pressing the number multiple times cycles through each window of a given app. This is part of my daily use of 60+ windows. 

**Describe the solution you'd like**
These keyboard shortcut seem to not be available in Cairo. Maybe there are others but I can't find any. 
Would it be possible to implement that?

**Comment by Drexl** [comment]
Yep, this is the first thing I tried after installing Cairo. This and being able to manually order the taskbar items are essential features for me.

**Comment by lostmsu** [comment]
I just tried actually implementing this and my problem is that `HotKey` class refuses to register `Windows+[Number]` global hook because the hotkey is already registered (at least when not running as shell). Which seems weird to me because Cairo replaces the system taskbar and the system taskbar no longer appears to be handling those combinations.

**Comment by lostmsu** [comment]
Heads up, the hotkey registration works via [MouseKeyHook](https://www.nuget.org/packages/MouseKeyHook/5.6.0)
